dora/core/common/src/main/java/alluxio/util/UpdateCheckUtils.java

```diff
@@ -119,7 +114,6 @@ public static String getUserAgentString(String id, List<String> additionalInfo)
     List<String> info = new ArrayList<>();
     info.add(id);
     addUserAgentEnvironments(info);
-    addUserAgentFeatures(info);
```

Comment:
is this removal on purpose? i do not see these user agent features readded for the master side update check

Reply:
yes, this is intentional. original `addUserAgentFeatures` method is only covering master features and none of them is relevant after 300. Now, if we need any feature related info, make it in their respective UpdateChecker